src/views/modal-view.js

```diff
@@ -0,0 +1,30 @@
+define([
+    'streamhub-sdk/modal/views/attachment-gallery-modal',
+    'inherits'
+], function (
+    AttachmentGalleryModal,
+    inherits
+) {
+    'use strict';
+
+    var MapModalView = function(opts) {
+        AttachmentGalleryModal.call(this, opts);
+    };
+    inherits(MapModalView, AttachmentGalleryModal);
+
+    /**
+     * Overriding the hide method in the ModalView from the SDK to remove the
+     * "hiding" event because it removes the lf-package from the "hub-modals"
+     * element, which causes problems loading child modals.
+     * @override
+     */
+    MapModalView.prototype.hide = function () {
+        this.$el.hide();
+        this._detach();
+        this.visible = false;
+        $('body').css('overflow', 'auto');
```

Comment:
This is all overrides. I didn't do anything besides remove the trigger for hiding.

On Tue, Jul 7, 2015 at 12:00 PM, Benjamin Goering
<notifications@github.com> wrote:

>> +    var MapModalView = function(opts) {
>> +        AttachmentGalleryModal.call(this, opts);
>> +    };
>> +    inherits(MapModalView, AttachmentGalleryModal);
>> +
>> +    /**
>> +     * Overriding the hide method in the ModalView from the SDK to remove the
>> +     * "hiding" event because it removes the lf-package from the "hub-modals"
>> +     * element, which causes problems loading child modals.
>> +     * @override
>> +     */
>> +    MapModalView.prototype.hide = function () {
>> +        this.$el.hide();
>> +        this._detach();
>> +        this.visible = false;
>> +        $('body').css('overflow', 'auto');
> This is depending on a global `$`? And what if the customer doesn't want the body overflow to be auto?
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/Livefyre/streamhub-map/pull/22/files#r34077179